iOS/src/toga_iOS/widgets/button.py

```diff
@@ -68,10 +68,9 @@ def set_color(self, color):
             )
 
     def set_background_color(self, color):
-        if color == TRANSPARENT or color is None:
-            self.native.backgroundColor = None
-        else:
-            self.native.backgroundColor = native_color(color)
+        super().set_background_color(
+            self._default_background_color if color in {None, TRANSPARENT} else color
+        )
```

Comment:
AFAICT, this is a change in behavior. `backgroundColor=None` is a required call in this instance, and this new implementation doesn't provide that as an option.

Reply:
I've restored it.